### PR TITLE
Use @comunica/actor-http-solid-auth-fetch for HTTP

### LINKED ISSUE #6: Use solid.auth.fetch to retrieve files

### PR DESCRIPTION
**Pull request by joachimvh**
This resolves #6

### CODE REVIEW

File: config/sets/http-solid.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          "@id": "config-sets:http.json#myHttpFetcher",
+          "@type": "ActorHttpSolidAuthFetch"
+        }
+      ]
+    },
+    {
+      "@id": "config-sets:http.json#mediatorHttp",
+      "@type": "MediatorNumberMin",
+      "field": "time",
+      "ignoreErrors": true,
```
Comment (RubenVerborgh) [inline config/sets/http-solid.json]: meaning?

Comment (rubensworks) [reply config/sets/http-solid.json]: This is the same as our default HTTP mediator. This means that if multiple actors are in the bus, and one of them rejects a test, the next one will be checked. Only if all actors reject, then the mediator will reject as well.

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@comunica/actor-init-sparql": "^1.5.0",
     "@comunica/actor-rdf-resolve-quad-pattern-rdfjs-source": "^1.5.4",
+    "@comunica/actor-http-solid-auth-fetch": "^1.0.0",
     "mkdirp": "^0.5.1"
```
Comment (rubensworks) [inline package.json]: Not caused by this PR, but I assume `mkdirp` should be a dev dependency.

Comment (RubenVerborgh) [reply package.json]: Will fix.

**Pull request merged**